src/io/files.js

```diff
@@ -1091,35 +1104,41 @@ p5.prototype.httpDo = function() {
     }
   }
 
-  (type === 'jsonp' ? fetchJsonp(path, jsonpOptions) : fetch(request))
-    .then(function(res) {
-      if (!res.ok) {
-        var err = new Error(res.body);
-        err.status = res.status;
-        err.ok = false;
-        throw err;
-      }
+  var promise;
+  if (type === 'jsonp') {
+    promise = fetchJsonp(path, jsonpOptions);
+  } else {
+    promise = fetch(request);
+  }
+  promise = promise.then(function(res) {
+    if (!res.ok) {
+      var err = new Error(res.body);
+      err.status = res.status;
+      err.ok = false;
+      throw err;
+    }
```

Comment:
Can you explain a bit about why the style change here? Not that I don't like it or something like that but rather I believe @Spongman recently changed it to the one line style and I don't want this to keep switching around as different people work on the codebase. Thanks.

Reply:
```javascript
var promise = (type === 'jsonp' ? fetchJsonp(path, jsonpOptions) : fetch(request))
  .then(...)
```
fell fowl of the linter for having too long a line - I could either break the ternary operator across multiple lines or split it into an if with a variable. As a variable was needed later anyway to return the correct promise (Before the `.then` and `.catch`, so that the errors are propagated forward correctly), this seemed like the cleanest way to do it to me.